docs/Tracking model classes list.md

```diff
@@ -0,0 +1,14 @@
+# Available tracking events model classes
+* VSDKAddToCart
+* VSDKPageView
```

Comment:
What do you think about having links like this?

```suggestion
* [VSDKPageView](Classes/VSDKPageView.html)
```

Reply:
What about like this 078a496?

Reply:
👍